src/include/dav1d/dav1d.rs

```diff
@@ -142,8 +143,8 @@ pub struct Dav1dSettings {
 
 #[repr(C)]
 pub(crate) struct Rav1dSettings {
-    pub n_threads: c_int,
-    pub max_frame_delay: c_int,
+    pub n_threads: u32,
+    pub max_frame_delay: u32,
     pub apply_grain: bool,
     pub operating_point: u8,
```

Comment:
Also, to make the API more explicit (and in case it's used directly from Rust), could you  make these fields `n_threads: InRange<u16, 0, 256>`, `max_frame_delay: InRange<u16, 0, 256>`, and `operating_point: InRange<u8, 0, 31>`?